contracts/EventBasedPredictionMarket.sol

```diff
@@ -7,17 +7,8 @@ import "@uma/core/contracts/common/implementation/ExpandedERC20.sol";
 import "@uma/core/contracts/common/implementation/Testable.sol";
 import "@uma/core/contracts/oracle/implementation/Constants.sol";
 
-import "@uma/core/contracts/oracle/interfaces/FinderInterface.sol";
-import "@uma/core/contracts/oracle/interfaces/OptimisticOracleInterface.sol";
-
-// TODO use OptimisticOracleInterface from @uma/core once it's updated with setEventBased
-interface OptimisticOracleInterfaceEventBased {
-    function setEventBased(
-        bytes32 identifier,
-        uint256 timestamp,
-        bytes memory ancillaryData
-    ) external;
-}
+import "./oracle/interfaces/OptimisticOracleV2Interface.sol";
```

Comment:
we will be able to replace this shortly :)

Reply:
actually, I'm confused. where is this `OptimisticOracleV2Interface.sol` file? it might make sense for us to wait until we've done the next release before merging this PR